vita3k/kernel/include/kernel/object_store.h

```diff
@@ -49,7 +49,7 @@ class ObjectStore {
     }
 
     template <typename T, typename... Args>
-    bool create(Args &&... args) {
+    bool create(Args &&...args) {
```

Comment:
hm

Reply:
Unrelated format changes not to be here.

Reply:
is done by format bat ^^

Reply:
bbb vv zz